### PR TITLE
Readme changes for account linking sample

### PR DESCRIPTION
**Pull request by Prithvi-MSFT**
Added images and few readme changes for account linking sample

### CODE REVIEW

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving it.

**Review by Wajeed-msft — APPROVED** [review]

File: samples/account-linking/csharp/README.md [hunk 1]
```diff
@@ -98,8 +100,16 @@ Please fill in the following values into the `Manifest/Manifest.json` file (call
 | Bot Id  | The "Application (client) id" from step 3. |
 | Tab domain | The domain from step 1 |
```
Comment (beauzeaux) [inline samples/account-linking/csharp/README.md:101]: If we're going add the `Base Url` `Base Url Domain` we don't need the `Tab domain` (you removed this from the manifest.json and replaced it with `Base Url`)

**Review by beauzeaux — APPROVED** [review]
LGTM

File: samples/account-linking/csharp/README.md (hunk 1 above)
Comment (Prithvi-MSFT) [reply samples/account-linking/csharp/README.md]: Yes removed it

**Pull request merged**